qiskit/test/mock/backends/armonk/conf_armonk.json

```diff
@@ -1 +1 @@
"name": "u2", "parameters": ["phi", "lambda"], "qasm_def": "gate u2(phi,lambda) q { U(pi/2,phi,lambda) q; }", "coupling_map": [[0]]}, {"name": "u3", "parameters": ["theta", "phi", "lambda"], "qasm_def": "gate u3(theta,phi,lambda) q { U(theta,phi,lambda) q; }", "coupling_map": [[0]]}], "local": false, "simulator": false, "conditional": false, "open_pulse": true, "memory": true, "max_shots": 8192, "coupling_map": null, "dynamic_reprate_enabled": false, "supported_instructions": ["setf", "measure", "x", "cx", "id", "u2", "shiftf", "acquire", "u1", "delay", "u3", "play"], "max_experiments": 75, "sample_name": "SPARROW", "n_registers": 1, "credits_required": true, "online_date": "2019-10-16T04:00:00+00:00", "description": "1 qubit device", "dt": 0.2222222222222222, "dtm": 0.2222222222222222, "allow_q_object": true, "multi_meas_enabled": false, "parametric_pulses": ["gaussian", "gaussian_square", "drag", "constant"], "quantum_volume": null, "qubit_channel_mapping": [["m0", "d0"]], "uchannels_enabled": true, "url": "None", "allow_object_storage": true, "n_uchannels": 0, "u_channel_lo": [], "meas_levels": [1, 2], "qubit_lo_range": [[4.474446855948306, 5.474446855948306]], "meas_lo_range": [[6.493427855, 7.493427855]], "meas_kernels": ["hw_boxcar"], "discriminators": ["linear_discriminator", "quadratic_discriminator"], "rep_times": [1000.0], "meas_map": [[0]], "acquisition_latency": [], "conditional_latency": [], "hamiltonian": {"description": "Qubits are modeled as Duffing oscillators. In this case, the system includes higher energy states, i.e. not just |0> and |1>. The Pauli operators are generalized via the following set of transformations:\n\n$(\\mathbb{I}-\\sigma_{i}^z)/2 \\rightarrow O_i \\equiv b^\\dagger_{i} b_{i}$,\n\n$\\sigma_{+} \\rightarrow b^\\dagger$,\n\n$\\sigma_{-} \\rightarrow b$,\n\n$\\sigma_{i}^X \\rightarrow b^\\dagger_{i} + b_{i}$.\n\nQubits are coupled through resonator buses. The provided Hamiltonian has been projected into the zero excitation subspace of the resonator buses leading to an effective qubit-qubit flip-flop interaction. The qubit resonance frequencies in the Hamiltonian are the cavity dressed frequencies and not exactly what is returned by the backend defaults, which also includes the dressing due to the qubit-qubit interactions.\n\nQuantities are returned in angular frequencies, with units 2*pi*GHz.\n\nWARNING: Currently not all system Hamiltonian information is available to the public, missing values have been replaced with 0.\n", "h_latex": "\\begin{align} \\mathcal{H}/\\hbar = & \\sum_{i=0}^{0}\\left(\\frac{\\omega_{q,i}}{2}(\\mathbb{I}-\\sigma_i^{z})+\\frac{\\Delta_{i}}{2}(O_i^2-O_i)+\\Omega_{d,i}D_i(t)\\sigma_i^{X}\\right) \\\\ \\end{align}", "h_str": ["_SUM[i,0,0,wq{i}/2*(I{i}-Z{i})]", "_SUM[i,0,0,delta{i}/2*O{i}*O{i}]", "_SUM[i,0,0,-delta{i}/2*O{i}]", "_SUM[i,0,0,omegad{i}*X{i}||D{i}]"], "osc": {}, "qub": {"0": 3}, "vars": {"delta0": -2.189518576306432, "omegad0": 0.0721186503632053, "wq0": 31.25537139664009}}}
```

Comment:
might we send these through a JSON pretty printer?

Reply:
probably too much to ask

Reply:
It's not really worth it, these are just raw dumps of the api response from the ibmq api they're not meant to be read by humans. We want to preserve the raw formatting because they're only committed to be parsed directly by the fake backends. Any formatting we do will just make the files larger (and they're already quite big)